workorders.lic

```diff
@@ -719,6 +728,64 @@ class WorkOrders
     fput('stow deed')
   end
 
+  def order_enchant(stock_room, stock_needed, stock_number)
+    stock_needed.times do
+      order_item(stock_room, stock_number)
+      stow_tool(left_hand)
+      stow_tool(right_hand)
+      next unless left_hand && right_hand
+    end
+  end
+
+  def enchanting_items(info, recipe, quantity)
+    ensure_copper_on_hand(@cash_on_hand || 20000, @settings)
+    # Enchant Sigil #1
+    order_enchant(info['stock-room'], quantity, recipe['enchant_stock1'])
+    
+    # Enchant Sigil #2
+    order_enchant(info['stock-room'], quantity, recipe['enchant_stock2']) unless recipe['enchant_stock2'].nil?
+    
+    # Sigil 3
+    order_enchant(info['stock-room'], quantity, recipe['enchant_stock3']) unless recipe['enchant_stock3'].nil?
+    
+    # Enchant component #1
+    order_enchant(info['stock-room'], quantity, recipe['item']) unless recipe['item'].nil?
+
+    # Parts #
+    order_parts(recipe['part'], quantity) if recipe['part']
+
+    # Check for fount #
+    if search?('fount')
+      waitrt?
+    case bput('get my fount', 'You get', 'What were you referring to')
+      when 'You get'
+        /(\d+)/ =~ bput('focus my fount', 'Judging from the intensity of the glow within you estimate it has \d+ uses remaining')
+        if Regexp.last_match(1).to_i < quantity
+          dispose('fount')
+          stow_hands
+          order_enchant(info['tool-room'], 1, info['fount'])
+        end
+      when 'What were you referring to'
+        /(\d+)/ =~ bput('focus my fount on my brazier', 'Judging from the intensity of the glow within you estimate it has \d+ uses remaining')
+        if Regexp.last_match(1).to_i < quantity
+          stow_hands
+          order_enchant(info['tool-room'], 1, info['fount'])
+        end
+    end
+    else
+      order_enchant(info['tool-room'], 1, info['fount'])
+    end
+    stow_hands
+
+    # Start crafting #
+    find_enchanting_room(@hometown, @enchanting_room)
```

Comment:
💡 I prefer to see this as `DRCC.find_enchanting_room` (same for the other common functions)